webapp-client/src/stylesheets/Category.css

```diff
@@ -0,0 +1,5 @@
+.category-name {
+  font-size: 25px;
+  background-color: aquamarine;
+  text-align: center;
```

Comment:
bulma "has-text-centered" instead?

Reply:
Both Category.css and Category.js will be replaced by Anthony's ItemContainer component #15 , which is why Category component is still the placeholder version without bulma styling:)